Formula/butteraugli.rb

```diff
@@ -0,0 +1,20 @@
+class Butteraugli < Formula
+  desc "Measuring differences between images."
+  homepage "https://github.com/google/butteraugli"
+  url "https://github.com/google/butteraugli/archive/2f8d6a0b1466b003d610271ef1f0ae3dd0d28998.tar.gz"
+  version "0.1"
```

Comment:
This looks somewhat arbitrary, but I guess that's because upstream doesn't provide any tagged releases? I think we'll need to find something better to put here. (If interested, check out the discussion in #1256, particularly the now hidden code comments, for some context.)